src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -1821,7 +1821,7 @@ Show_extra_columns=Montrer_les_colonnes_supplémentaires
 Parsing_error=Erreur_de_traitement
 illegal_backslash_expression=Expression_avec_barre_oblique_inversée_(\\)_illégale
 
-Move_to_group=Déplacer_vers_le_groupe
+Move_to_group=
```

Comment:
Sorry. I think this problem is happened, during i merge and assign gradle update 😄. But i have redone it 👍